openmls/tests/test_group.rs

```diff
@@ -454,8 +453,12 @@ fn group_operations(ciphersuite: &'static Ciphersuite, backend: &impl OpenMlsCry
         Err(e) => panic!("Error creating commit: {:?}", e),
     };
 
-    // Check that there is a new KeyPackageBundle
-    assert!(create_commit_result.key_package_bundle_option.is_some());
+    // Check that there is a path
+    let commit = match create_commit_result.commit.content() {
+        MlsPlaintextContentType::Commit(commit) => commit,
+        _ => panic!("Wrong content type"),
```

Comment:
I'm never too sure if `panic!` in a branching statement isn't throwing codecov off course.
The alternative would be to return an `Option` that then gets unwrapped with an `expect()`. I don't have a strong opinion though.

Reply:
Only one way to find out. I removed a bunch of panics. Before, we had `Coverage   86.92%   87.08%   +0.15%` according to codecov..

Reply:
Looks like it actually got worse: to the point that codecov is not green anymore: `Coverage   86.92%   87.02%   +0.09%` 🤷🏻‍♂️

Reply:
Thanks for trying! At least now we know. Do you want to roll back the changes?

Reply:
Why not. I think at this point it doesn't really matter.